enhancements/etcd/automated-backups-no-config.md

```diff
@@ -0,0 +1,299 @@
+
+The approach has been implemented and tested. Below are the Pros and Cons found.
+
+**Pros**
+
+As the container runs on the same pod as etcd, it requests backup through `localhost`. Backups are being stored on the `hostPath` volume, with is a Directory on the node's file system. This guarantees no network delays.
+
+**Pros**
+
+Etcd pod is a static pod, and the availability of the etcd cluster is vital for cluster stability. As the etcd cluster is being managed by `Cluster-etcd-operator`, and a `Deployment` resource manages the _mirrored_ etcd pods, any changes in the pod manifest results into rolling out a new revision, which leads to a duration of etcd-cluster unavailability. Consequently, the `API-Server` can not process any write requests, and read requests are being handled; if any, from the `API-Server` cache.   
+
+The backup sidecar container is being `disabled` by default. Upon creating a `Backup` CR on an Openshift cluster, the `PeriodicBackupController` reacts by enabling the container. This leads to a change in etcd static pod manifest, and a new revision is being rolled out. As explained above, such action causes cluster instability and `API-Server` unavailability.
+
+Additionally, any error within the backup sidecar, the container goes into `CrashLoopBackOff`, which leads to the whole `Pod` reports `NotReady` status. Consequently, the whole `Etcd-cluster` can not server requests, as well as `API-Server`.
+
+
+#### DaemonSet manages an Etcd-backup pod within every control-plane node.
+
+This approach has been initiated during reviews of the `Sidecar` method. A reviewer has recommended to separate the backup container from the static pod to mitigate the issues shown above.
+Initially a `Deployment` resource was suggested, before using a `DaemonSet`. The advantage of using DaemonSet over Deployment is to guarantee scheduling `Etcd-backup` pods on all `Control-plane` nodes, regardless of the number of nodes (e.g. 1, 3, 5) using `nodeSelector` capability.
+By setting the `nodeSelector` to the master node label (i.e. `node-role.kubernetes.io/master`), guarantees that an Etcd-backup pod will be scheduled on every control-plane node.
+
+**Pros**
+
+This method has been implemented and tested. The same container used in the Sidecar approach has been utilised. However, the scheduling is being managed by a DaemonSet resource.
+Upon creating a `Backup` CR on an Openshift cluster, the `PeriodicBackupController` reacts by creating a DaemonSet resource, with the `etcd-backup-server` container and the configurations applied in the CR.
+During [testing](https://github.com/openshift/cluster-etcd-operator/pull/1354#issuecomment-2408661261), this method has shown a solid and stable deployment, where the Backup pods shown only `1` restart over a duration of `45` minutes. Moreover, the cause of the restart on all the Backup pods were due to `node` issue, not the backup container.
+
+**Cons**
```

Comment:
yes, they are all being taken simultaneously

Reply:
And, when a backup occurs, does that prevent reads and/or writes to the etcd instance/cluster?

Reply:
gr8 Question. 

So I don't think so, citing https://etcd.io/docs/v3.5/op-guide/recovery/

```
Note that taking the snapshot from the member/snap/db file might lose data that has not been written yet, but is included in the wal (write-ahead-log) folder.
```

Read/Write request has to go through Raft and the WAL, since the backup relies on the snapshot, the backup might be lacking the records that have been replicated through the WAL, but not applied to the StateMachine yet (i.e. bbolt) 

cc @hasbro17 @dusk125 @tjungblu